### ISSUE TITLE
System.ComponentModel.Primitives for .NET 4.7

### ISSUE THREAD

**Issue by djekmusic** [dotnet/corefx#30754]

**Comment by danmosemsft** [comment]
@djekmusic this question does not relate to .NET Core, am I right?

I don't know why the warning though. A detailed build log might help you. Or @weshaggard may have an idea.

**Comment by djekmusic** [comment]
@danmosemsft yes, you are right - it's not .NET Core related, but Nuget support (@weshaggard ) asked me to submit question to this hub

**Comment by weshaggard** [comment]
@danmosemsft while it isn't related to .NET Core it is related to a package we used to ship out of corefx. It doesn't look like we ship it any longer but we support it here.

I don't see anything obvious in the package that would make it work differently between .NET 4.6.2 and .NET 4.7.1. 

@djekmusic are you referencing the 4.3.0 package? Are you actually using it in your application and if so does it work?

**Comment by djekmusic** [comment]
@weshaggard the problem is we don't use it directly. It's dependency of another library. That library works as well.
Would you suggest any kind of test to run? 

In assembly explorer I see only internal classes there (some resources).

**Comment by weshaggard** [comment]
To test ideally you would call some code that would cause it to execute something in that library but given this isn't a dependency of another library it might be hard to do that. 

In an attempt to try and fix the warning could you try referencing the System.ComponentModel.Primitives packages directly in your application and see if you still get the warning?
